### ISSUE TITLE
如何修改成简单的分类任务？

### ISSUE THREAD

**Issue by VincentZ-2020** [JDAI-CV/fast-reid#139]
譬如imagenet分类任务，和简单的top1，5的评价方式？

**Comment by L1aoXingyu** [comment]
可以自己写一个 `evaluator`，现在用的是 reid_evaluator.py，你可以改成分类的测试方式 https://github.com/JDAI-CV/fast-reid/blob/df8916f1e6ce5cdacbe93b37d7850a7ac720ebf6/fastreid/evaluation/reid_evaluation.py#L23

同时要改一下 `head`，因为现在是 reid 测试，所以 `head` 的输出都是一个 embedding，你可以改成 logits 的输出。
